changes/492.fix.md

```diff
@@ -0,0 +1 @@
+Raise an error on deleting vfolder by its name if there is the same name of vfolder in multiple accessible storage hosts.
```

Comment:
Suggestion:
> Do not delete a virtual folder if there are other folders with the same name (in other folder hosts), rather than blindly and dangerously deleting the first-queried one.